docs/source/node.rst

```diff
@@ -32,7 +32,9 @@ Once you have installed the executable, you can invoke it by running:
 
   $ livepeer
 
-Note: by default Livepeer listens to the local interface.  This means if you are running Livepeer on a cloud-hosted instance, you need to use additional flags ``--rtmpIP 0.0.0.0`` and ``--httpIP 0.0.0.0`` if you want it to listen to incoming network requests.
+Note: by default Livepeer listens to the local interface.  This means if you are running Livepeer on a cloud-hosted instance, you need to set the ``--rtmpIP 0.0.0.0`` flag.
```

Comment:
Does this only apply to "ingest" nodes used for broadcasting? Not for transcoders I assume?

Reply:
Transcoders don't start a RTMP listener, but they do start a HTTP listener, so they certainly should be careful about exposing HTTP.